rust/toolchain.bzl

```diff
@@ -71,26 +71,91 @@ def _make_libstd_and_allocator_ccinfo(ctx, rust_lib, allocator_library):
                 print("File partitioned: {}".format(f.basename))
             fail("rust_toolchain couldn't properly partition rlibs in rust_lib. Partitioned {} out of {} files. This is probably a bug in the rule implementation.".format(partitioned_files_len, len(dot_a_files)))
 
+    return [
+        DefaultInfo(
+            files = depset(ctx.files.srcs),
+        ),
+        RustStdLibInfo(
+            std_rlibs = std_rlibs,
+            dot_a_files = dot_a_files,
+            between_alloc_and_core_files = between_alloc_and_core_files,
+            core_files = core_files,
+            between_core_and_std_files = between_core_and_std_files,
+            std_files = std_files,
+            alloc_files = alloc_files,
+        ),
+    ]
+
+rust_stdlib_filegroup = rule(
```

Comment:
Please expose this rule through `//rust:defs.bzl`. That should be the main entry point for everything users need from rules Rust. The fact that the toolchain is not there yet is sad, but I do plan to move rust_toolchain there soon (definitely before 1.0).